test/e2e/images_test.go

```diff
@@ -112,7 +113,7 @@ var _ = Describe("Podman images", func() {
 		Expect(rmi.ExitCode()).To(Equal(0))
 
 		dockerfile := `FROM docker.io/library/alpine:latest
-`
+	   `
```

Comment:
Why the whitespace changes here?

Reply:
gofmt must have done it. maybe to match the above line? I'm not sure

Reply:
It's inside a string, so I doubt gofmt would touch it.
Also, I'm wondering why we even bothered with backticks here. `"FROM docker.io/library/alpine:latest\n"` is perfectly sufficient and fits on one line

Reply:
whitespace change removed!